charts/prometheus-postgres-exporter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "0.10.0"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 2.3.7
+version: 2.3.8
```

Comment:
New feature should raise minor version.

Reply:
Hello,
Sure i will set it to 2.4.0.
Thanks